Configuration/Skimming/interface/DisappearingMuonsSkimming.h

```diff
@@ -47,28 +49,28 @@ class DisappearingMuonsSkimming : public edm::one::EDFilter<> {
   static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
 
 private:
-  void beginJob() override;
   bool filter(edm::Event&, const edm::EventSetup&) override;
-  void endJob() override;
+
   bool passTriggers(const edm::Event& iEvent,
-                    edm::EDGetToken m_trigResultsToken,
+                    const edm::TriggerResults& results,
                     std::vector<std::string> m_muonPathsToPass);
```

Comment:
Pass also this as a reference?

Reply:
yes done in https://github.com/cms-sw/cmssw/commit/7a3a15df8d7e392f2e1394ca2120e5c5f2e6512c